code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -254,20 +254,38 @@ datum
 			penetrates_skin = 1
 			blob_damage = 5
 			value = 7 // 3 2 1 heat
+			var/counter = 1
+
+			pooled()
+				..()
+				counter = 1
 
 			on_mob_life(var/mob/M, var/mult = 1) // -cogwerks. previous version
 				if (!M) M = holder.my_atom
+				if (!counter) counter = 1
 				M.take_toxin_damage(1.5 * mult)
 				if (probmult(8))
 					M.emote("drool")
 				if (prob(10))
 					boutput(M, "<span class='alert'>You cannot breathe!</span>")
-					M.losebreath += (1 * mult)
+					M.losebreath += (mult)
 					M.emote("gasp")
-				if (prob(8))
-					boutput(M, "<span class='alert'>You feel horribly weak.</span>")
-					M.setStatus("stunned", max(M.getStatusDuration("stunned"), 30 * mult))
-					M.take_toxin_damage(2 * mult)
+				switch(counter+= (mult))
+					if (20 to 30)
+						if (prob(15))
+							boutput(M, "<span class='alert'>You feel weak.</span>")
+							M.setStatus("stunned", max(M.getStatusDuration("stunned"), 1 SECONDS * mult))
+							M.take_toxin_damage(0.5 * mult)
+					if (31 to 45)
+						if (prob(20))
+							boutput(M, "<span class='alert'>You feel very weak.</span>")
+							M.setStatus("stunned", max(M.getStatusDuration("stunned"), 2 SECONDS * mult))
+							M.take_toxin_damage(mult)
```

Comment:
```suggestion
							M.take_toxin_damage(1 * mult)
```

Reply:
So cutting the stuns in half would be a good idea? What would be your stance on making the loosebreath chance also go up with exposure time?

Reply:
Yes. Sorry, that was my bad. I spoke to Yass and agree with their take.